otter/convergence/service.py

```diff
@@ -162,7 +162,7 @@ def all_met(server, current_lb_nodes):
                              if node.matches(server)]))
 
 
-def update_cache(group, servers, lb_nodes, now):
+def update_cache(group, servers, lb_nodes, now, include_deleted=True):
```

Comment:
@manishtomar: The idea was that logically, whatever servers you give `update_cache` should be cached and if necessary, the deleted ones can be excluded. Therefore, I thought it made sense to give it a default. I am fine making it a positional argument, but what do you think about that?

And I will add a docstring.